src/esp/gfx/PbrShader.h

```diff
@@ -0,0 +1,453 @@
+// Copyright (c) Facebook, Inc. and its affiliates.
+// This source code is licensed under the MIT license found in the
+// LICENSE file in the root directory of this source tree.
+
+#ifndef ESP_GFX_PBRSHADER_H_
+#define ESP_GFX_PBRSHADER_H_
+
+#include <initializer_list>
+#include <vector>
+
+#include <Corrade/Containers/ArrayView.h>
+#include <Corrade/Containers/EnumSet.h>
+#include <Magnum/GL/AbstractShaderProgram.h>
+#include <Magnum/Math/Matrix4.h>
```

Comment:
*Technically* you don't need this include either as there's a forward declaration that's enough. 

OTOH, if *do* you want to have the full definition here, you'll need `Matrix3.h` as well for `setTextureMatrix()`.

Reply:
I do not need either of them in the .h file.